lib/gtfs_server/upload_parser/processor.ex

```diff
@@ -0,0 +1,131 @@
+defmodule GtfsServer.UploadProcessor do
+  NimbleCSV.define(CSVParser, separator: ",")
+  alias Utility
+  alias GtfsServer.Agencies.Agency
+  alias GtfsServer.Agencies.Route
+  alias GtfsServer.Agencies.Trip
+  alias GtfsServer.Agencies.Stop
+  alias GtfsServer.Agencies.StopTime
+  alias GtfsServer.Repo
+
+  import Ecto.Changeset
+  import GtfsServerWeb.ErrorHelpers, only: [translate_error: 1]
+
+  require Logger
+
+  @moduledoc """
+    UploadProcessor opens up the gtfs zip archive
+  """
+
+  def process(path) do
+    with {:ok, handle} <- get_zip_handle(path),
+      :ok <- process_agencies(handle),
+      :ok <- process_routes(handle),
+      :ok <- process_trips(handle),
+      :ok <- process_stops(handle)
+      # :ok <- process_stop_times(handle)
```

Comment:
When I try to process the stop times, the server seems to buckle under the pressure. This is the largest file in the zip archive (~165 mbs) and I don't think it is getting past opening the file.